src/Facing.php

```diff
@@ -159,17 +149,17 @@ public static function validate(int $facing) : void{
 	}
 
 	/**
+	 * @deprecated use Facing->name
```

Comment:
If this is a valid replacement, why isn't the function using it?

Reply:
Indead